app/models/collection_template.rb

```diff
@@ -63,6 +64,26 @@ def cleaned_image
     "#{fingerprinted_name}_tmp"
   end
 
+  def pathselected_image
+    "#{image.file_name_no_extension}_"\
+    "#{fingerprint_pathselection}_tmp"
+  end
+
+  def fingerprint_pathselection
+    "#{fingerprinted_name}_"\
+    "#{image_paths_to_hex.join('_')}"
+  end
+
+  def image_paths_to_hex
+    image_paths.map do |image_path|
+      # split on commas
+      image_path.split(',')
+                .map { |value| value.to_i.to_s(16).rjust(2, '0') }.join('')
+      # convert string to int to hex string and justify
+      # http://blog.lebrijo.com/converting-rgb-colors-to-hexadecimal-with-ruby/
```

Comment:
Hmm yeah that might be cool. Let me know if the CLI changes and we can probably make that change on our side.